client/components/thank-you-v2/footer/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, Gridicon } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { isOutsideCalypso } from 'calypso/lib/url';
+
+import './style.scss';
+
+export type ThankYouDetailProps = {
+	title?: string;
+	description?: string;
+	buttonText?: string;
+	href?: string;
+	clickEventName?: string;
+};
+
+const ThankYouDetail = ( {
+	title,
+	description,
+	buttonText,
+	href,
+	clickEventName,
+}: ThankYouDetailProps ) => {
+	const handleClick = () => {
+		if ( ! clickEventName ) {
+			return;
+		}
+
+		recordTracksEvent( clickEventName );
+	};
+
+	const isExternal = isOutsideCalypso( href ?? '' );
+
+	return (
+		<div className="thank-you__detail">
```

Comment:
I'm going to use the prefix `thank-you__footer-detail` prefix instead of `thank-you__footer_detail` following this convention: https://en.bem.info/methodology/naming-convention/#two-dashes-style

Reply:
Oh yes, I shouldn't have use an underscore in that particular place 👍.